### ISSUE TITLE
When running open world, we can add, but not remove constellations

### ISSUE THREAD

**Issue by CerielJacobs** [NLeSC/Constellation#15]
... but we should be able to remove them as well.

**Comment by jmaassen** [comment]
Removing constellations leads to a big can of worms with lost jobs, unroutable events, etc. That's why I never dared to implement it ;-)

**Comment by CerielJacobs** [comment]
True. It sounds very tricky indeed, especially with event routing. It is probably doable to stop executors from stealing jobs, and waiting until all activities are done (although that, by itself, might cause deadlocks...).
